src/test/shell/bazel/BUILD

```diff
@@ -1343,3 +1343,13 @@ sh_test(
     tags = ["no_windows"],
     deps = ["@bazel_tools//tools/bash/runfiles"],
 )
+
+sh_test(
+    name = "verify_workspace",
+    srcs = ["verify_workspace.sh"],
+    data = [
```

Comment:
You can add the `"requires-network"` tag to enable network access for this test.

Reply:
Thanks, added that and "no-windows", since there's not really any benefit to running this on every platform.